mass_arm: [2,0,0,0,0,0,
           0,2,0,0,0,0,
           0,0,2,0,0,0,
           0,0,0,2,0,0,
           0,0,0,0,2,0,
           0,0,0,0,0,2]
damping_arm: [10,0,0,0,0,0,
              0,10,0,0,0,0,
              0,0,10,0,0,0,
              0,0,0,10,0,0,
              0,0,0,0,10,0,
              0,0,0,0,0,10]
stiffness_coupling: [10,0,0,0,0,0,
                     0,10,0,0,0,0,
                     0,0,20,0,0,0,
                     0,0,0,10,0,0,
                     0,0,0,0,10,0,
                     0,0,0,0,0,10]

base_link: "base_link"
end_link: "ee_link"
# [position (x, y, z)  orientation (x, y, z, w)]
desired_pose:  [0.0,-0.85,0.3,       0.0, -0.707, 0.00, 0.707]

workspace_limits: [-1.50, 1.50, 1.25, 1.80, 1.30, 1.75]
arm_max_vel: 1.5
arm_max_acc: 5.0
